api/pkg/apps/cms/types.go

```diff
@@ -33,6 +37,19 @@ func (c *CaseType) String() string {
 	return c.Name
 }
 
+func (c *CaseType) UnmarshalFormData(values url.Values) error {
+	c.Name = values.Get("name")
+	c.PartyTypeID = values.Get("partyTypeId")
+	c.TeamID = values.Get("teamId")
+	templateString := values.Get("template")
```

Comment:
Can we unmarshal the whole CaseTemplate object rather than unmarshaling the array of CaseTemplateFormElements? That would allow us to put some more config in the textbox, rather than just the inner array of formElements

eg `json.Unmarshal([]byte(templateString), &c.CaseTemplate)`

Reply:
Yeah, I just thought it was cumbersome to have to wrap every template in a `formElements` containing field, but if we're going to have more root-level items, I agree with you